### DISCUSSION TITLE
N00b trying to get AMP-dockerized to work with Casaos

### DISCUSSION THREAD

**Discussion by tantgron1337** [MitchTalmadge/AMP-dockerized#159] [Q&A]
As topic; I am trying to get AMP-dockerized to work with Casaos, and have little to none experience with Docker Compose.

But I have followed the eminent "Configuration" section and it installs, but complains that:

Error response from daemon: Cannot start container f6223cc39xxxeeb9c7763dcfcb9462e1f73552706c8315166eab8ead98691d9: address 02:42:AC:XX:XX:XX: invalid MAC address

I have generated the new MAC address as it should be fine, but I think I have tried to write it in the wrong place?

I´m sure that Iḿ not doing the "image" part right (For Docker Compose, use the following key next to image).

Could someone please explain where that "image" should be written? Or point me in the right direction where to learn that for Docker if its elementary?

I also want to thank Mitch for a great job doing this, THANKS!! :)

**Comment by MitchTalmadge** [comment]
Hey! Thanks for reaching out! Can you put your docker compose file here so I can check? The only thing you'll need to hide is the license and any passwords. 

Also, is the MAC address exactly `02:42:AC:XX:XX:XX` with the X's or did you hide some of it? If it does have the X, that is not a valid MAC address. It should only be numbers 0-9 and letters A-F

:)

**Comment by tantgron1337** [comment]
To start with; I'm so sorry not responding back to your kind reply! Thanks alot! :)

My intention is to get AMP running on Casaos that I would call a webgui for Docker (casaos.io or try it live at demo.casaos.io (casaos/casaos)). 

When I read your message I understood that I had to edit one of the  ´example compose.yml´ by hand and found where to paste the mac address. :D

It still won't start, I still got something wrong and it's me lacking the knowledge to handle the problem. :D

I also understand that my problem is casaos-specific, so I maybe have to get info some where else.


But still, I'm greatful for your time and keep up the good work!
